app/views/software_choices_filter.scala.html

```diff
@@ -153,8 +150,7 @@ <h1 class="govuk-heading-xl">@messages("softwareChoices.filter.title")</h1>
                     <button type="submit" class="govuk-button" data-module="govuk-button">@messages("softwareChoices.filter.filterResults")</button>
                 </noscript>
             </div>
-
-            <div id="provider-table-container" class="govuk-grid-column-two-thirds">
+                <div id="provider-table-container" class="govuk-grid-column-one-half govuk-form-group">
```

Comment:
if you use class="govuk-!-grid-column-one-half govuk-form-group" it makes the right size but in the wrong place